changelog.txt

```diff
@@ -90,7 +90,7 @@ Version 3.2.0
 - [MINOR] Implement an in-memory cache to avoid multiple decryptions of shared preferences (#1254)
 - [PATCH] Fixes deprecated NetworkInfo class (#847)
 - [MINOR] Device PoP keys are now generated with attestation flags and expose a certificate chain getter (#1247)
-- [PATCH] Fix MSAL-CPP injecting javascript loop (#1238)
+- [PATCH] Fix MSAL-CPP injecting javascript loop (#1238, #1490)
```

Comment:
This needs a new changelog entry for the version that will be released, not version 3.2, where it will not be fides.

Reply:
Added new changelog entry.